lib/mozart_fetcher/component.ex

```diff
@@ -67,7 +67,6 @@ defmodule MozartFetcher.Component do
   end
 
   defp metric(id, endpoint, status) when is_integer(status) do
-    ExMetrics.increment("error.component.process")
     ExMetrics.increment("error.component.process.#{id}")
```

Comment:
```suggestion
    ExMetrics.increment("error.component.process.#{id}")
    ExMetrics.increment("http.component.retry")
```

Reply:
I added it to the http_client.ex file. Isn't the component the wrong place as we won't know if it's a retry or not?

Reply:
sorry, for some reason i thought this was in the `config/metrics.exs` file